app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -140,14 +141,14 @@
         filters.push({predicate: predicate, value: filter});
       }
 
-      _.each(searchTerms, function(term){
-        if(term.indexOf('is:') >=0) {
+      _.each(searchTerms, function(term) {
```

Comment:
Yeah, I've thought about it, but it just puts up another barrier to contribution. I'll just fix stuff as I see it.